src/components/position.js

```diff
@@ -1,8 +1,8 @@
-var coordinatesMixin = require('../utils/coordinates').componentMixin;
 var registerComponent = require('../core/component').registerComponent;
-var utils = require('../utils/');
 
-module.exports.Component = registerComponent('position', utils.extend({
+module.exports.Component = registerComponent('position', {
+  type: 'coordinate',
```

Comment:
Assuming most components' update functions are just going to do this...

```javascript
var position = new THREE.Vector3(data.position.x, data.position.y, data.position.z);
```

... would it be possible for the `v3/vec3/vector3` type to parsed to a THREE.Vector3 instance, to avoid re-instantiating it in the render loop?